pipelines/installing-pipelines.adoc

```diff
@@ -6,6 +6,13 @@ include::modules/pipelines-document-attributes.adoc[]
 
 toc::[]
 
+This guide walks cluster administrators through installing the {pipelines-title} Operator to an {product-title} cluster.
```

Comment:
`through installing` -> `through the process of installing` ?

Reply:
fixed